ConformanceExamples/Package.swift

```diff
@@ -0,0 +1,15 @@
+// swift-tools-version:5.1
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+  name: "ExamplesProtocolConformance",
+  products: [],
+  dependencies: [],
+  targets: [
+    .testTarget(name: "Test1.5.2"),
```

Comment:
Let's not use section numbers to identify these things as they're highly unstable, especially in early stages of drafting.  This is why the C++ standard has these names like [dcl.dcl] for each section.  But I think it's too early for that, even.  I suggest we name things after chapter + section titles